cluster-scope/overlays/nerc-ocp-prod/ingresscontrollers/external-apps-ingress-controller.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: external-apps-ingress-controller
+  namespace: openshift-ingress-operator
+spec:
+  domain: apps.shift.nerc.mghpcc.org
+  defaultCertificate:
+    name: external-apps-ingress-certificate
```

Comment:
where's this certificate created? I don't see it in #47

Reply:
This cert does not yet exist, once I have it I'll update this PR

Reply:
I've added the secret into https://github.com/OCP-on-NERC/nerc-ocp-config/pull/47